_data/devs.yml

```diff
@@ -168,3 +168,12 @@
     - python
     - golang
   grad: 2022
+
+- github: 'https://github.com/vishal-android-freak'
+  linkedin: 'https://linkedin.com/in/vishalaf'
+  twitter: 'https://twitter.com/naa_rang'
+  skills:
+    - fullstack
+    - IoT
+    - strategy
```

Comment:
comeon, this is a excluding criteria, wtf 😂😂😂

Reply:
What is? 😅